watcher/lib/constants/constants.go

```diff
@@ -79,6 +79,8 @@ const (
 	// DurationLong is the duration of "long" retention policy in format InfluxDB expects
 	DurationLong = "52w"
 
+	// DefaultInterval is the aggregation interval for "default" retention policy
+	DefaultInterval = "1m"
```

Comment:
Shouldn't this be called `IntervalDefault` to keep naming consistent (contrast with `IntervalMedium` below)?